cheetah/accelerator.py

```diff
@@ -39,6 +39,8 @@ class Element(ABC, nn.Module):
     :param name: Unique identifier of the element.
     """
 
+    length: torch.Tensor = torch.zeros((1))
```

Comment:
So, I don't really like the fact that length is a class property and how the lengthless classes assign `length` after being initialised in `broadcast`.

**BUT** I explored an alternative, where `length` is a parameter of `Element.__init__` or a read-only property of `Element`. However, this doesn't play nicely with the `length` property in `Segment` or the writable `length` properties in other elements that actually have a length.

So I think for now this is a good and simple solution. Most likely, these things will change again when we start integrating automatic broadcasting functionality for convenience.

Reply:
Just thought I would leave this here for later reference. This is no call to action.